h2o-core/src/main/java/water/util/GAUtils.java

```diff
@@ -13,16 +13,40 @@
  */
 public class GAUtils {
   public static void logRequest(String uri, Properties header) {
+    boolean logged = logRequest2(uri, header);
+
+    final boolean debug = false;
+    if (debug && logged) {
+      System.out.println("GA LOGGED:  " + uri);
+    }
+    else {
+      System.out.println("GA DROPPED: " + uri);
```

Comment:
Isn't this going to pollute the output?

Maybe we can use logger and enable it for the trace level?

Reply:
No, debug is final false.
Once we're happy with it we can delete that block entirely.

Reply:
Because debug is "false" it will always print "GA DROPPED" regardless if it was dropped or not

Reply:
sorry, you're right, i fixed it.